recipe/meta.yaml

```diff
@@ -77,3 +77,4 @@ extra:
     - chohner
     - jakirkham
     - ocefpaf
+    - matthiasdiener
```

Comment:
Can you put your name in sorted alphabetically please? 

A

Reply:
Sure!